HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -2568,16 +2568,21 @@ def self.get_space_ua_values(space_type, weather)
 
     # Infiltration UA
     infiltration_cfm = nil
+    ach = nil
     if [HPXML::LocationCrawlspaceVented, HPXML::LocationAtticVented].include? space_type
       # Vented space
       if space_type == HPXML::LocationCrawlspaceVented
         vented_crawl = @hpxml.foundations.select { |f| f.foundation_type == HPXML::FoundationTypeCrawlspaceVented }[0]
         sla = vented_crawl.vented_crawlspace_sla
       else
         vented_attic = @hpxml.attics.select { |f| f.attic_type == HPXML::AtticTypeVented }[0]
-        sla = vented_attic.vented_attic_sla
+        if not vented_attic.vented_attic_sla.nil?
+          sla = vented_attic.vented_attic_sla
+        else
+          ach = vented_attic.vented_attic_ach
+        end
       end
-      ach = Airflow.get_infiltration_ACH_from_SLA(sla, 8.202, weather)
+      ach = Airflow.get_infiltration_ACH_from_SLA(sla, 8.202, weather) if ach.nil?
```

Comment:
@shorowit I figured out that this is actually not a bug. This method `calculate_space_design_temps`->`get_space_ua_values` will never be called when space_type = HPXML::LocationAtticVented. Its space design temperatures are calculated/defined in another methodology. For this `get_space_ua_values`, however, I am not sure whether it will be used in scope out of `calculate_space_design_temps` in future, so I retained its logic for `HPXML::LocationAtticVented` and added ach just in case.